soc/riscv/riscv-privilege/telink_b91/start.S

```diff
@@ -31,11 +31,13 @@ entry:
 
 start:
 
+#if !CONFIG_TELINK_B91_USB_SWI_ENABLE
```

Comment:
#ifndef CONFIG_TELINK_B91_USB_SWI_ENABLE
As I remember in case of KConfig boolean option is set to n generated .config doesn't provide this definition. Plese check it.

Reply:
Yes, I thought the same previously (and it works this way if default value from Kconfig is used), but what in case CONFIG_TELINK_B91_USB_SWI_ENABLE will be defined by user in prj.conf file as CONFIG_TELINK_B91_USB_SWI_ENABLE=n? 

Then CONFIG_TELINK_B91_USB_SWI_ENABLE is defined, but set to n and this will cause that ifndef will not work here.
We should check the value too.

Maybe something like this will be the best solution then?
#if !defined(CONFIG_TELINK_B91_USB_SWI_ENABLE) || !CONFIG_TELINK_B91_USB_SWI_ENABLE